src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_C0ST = 40;
+
+  const LONG_TERM = 7;
+  const LONG_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_DISCOUNT = 20;
+
+  let allResult = 0;
+
+  if (days >= LONG_TERM) {
+    allResult += RENT_CAR_C0ST * days - LONG_DISCOUNT;
+
+    return allResult;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
The condition 'days >= SHORT_TERM && days < LONG_TERM' can be simplified to 'days >= SHORT_TERM' since the previous 'if' condition already checks for 'days >= LONG_TERM'.